src/core/store/movie/actions.ts

```diff
@@ -0,0 +1,3 @@
+export default {
```

Comment:
These are just objects that store "actionTypes"and we usually group them based on business value.
To sum up considering the above comment:
"action" - object with only "type" and "payload" properties.
"actionType" - key:value pair that is used in "action" object.

We use "actionTypes" across the app to specify at any point what "action" we want to trigger. Think of these 2 constructs as browser "event" object and their respective "event" names like "click", "DOMContentLoaded", etc.